tests/phpunit/api/v3/EventTest.php

```diff
@@ -82,9 +92,6 @@ public function setUp(): void {
   }
 
   public function tearDown(): void {
-    foreach ($this->eventIds as $eventId) {
-      $this->eventDelete($eventId);
```

Comment:
This code was never being reached as `eventIds` was the _wrong_ property. It's also unnecessary as the event and participant tables are truncated a couple of lines down!

Reply:
🤦